cypress/e2e/ui/Settings/Application-Settings/c_and_u_gap_collection.cy.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable no-undef */
+import { flashClassMap } from '../../../../support/assertions/assertion_constants';
+
+// Menu options
+const SETTINGS_MENU_OPTION = 'Settings';
+const APP_SETTINGS_MENU_OPTION = 'Application Settings';
+
+// List items
+const DIAGNOSTICS_ACCORDION_ITEM = 'Diagnostics';
+const MANAGEIQ_REGION_ACCORD_ITEM = /^ManageIQ Region:/;
+const ZONE_ACCORD_ITEM = /^Zone:/;
+
+// Field values
+const TIMEZONE_FIELD_LABEL = 'Timezone';
+const START_DATE_FIELD_LABEL = 'Start Date';
+const END_DATE_FIELD_LABEL = 'End Date';
+const FORM_HEADER = 'Diagnostics Zone';
+const FORM_SUBHEADER_SNIPPET = 'Collection Options';
+const TIMEZONE_TYPE_HAWAII = '(GMT-10:00) Hawaii';
+const START_DATE = '06/25/2026';
+const END_DATE = '06/28/2026';
+
+// Buttons
+const SAVE_BUTTON_TEXT = 'Save';
+
+// Flash message text snippets
+const FLASH_MESSAGE_GAP_COLLECTION_INITIATED = 'initiated';
+const FLASH_MESSAGE_DATE_RANGE_INVALID = 'cannot';
+
+function fillGapCollectionForm(startDateValue, endDateValue) {
+  // Select "Hawaii" from timezone dropdown
+  cy.getFormLabelByInputId('timezone').click();
+  cy.contains('[role="option"]', TIMEZONE_TYPE_HAWAII).click();
+  // Add start date
+  cy.getFormInputFieldById('startDate').type(startDateValue);
+  // Add end date
+  cy.getFormInputFieldById('endDate').type(endDateValue, {
+    force: true,
+  });
+}
+
+function saveFormAndAssertFlashMessage(flashMessageType, flashMessage) {
+  cy.interceptApi({
+    alias: 'saveGapCollectionApi',
+    urlPattern: '/ops/cu_repair?button=submit',
+    triggerFn: () =>
+      cy
+        .getFormFooterButtonByType(SAVE_BUTTON_TEXT, 'submit')
```

Comment:
https://github.com/ManageIQ/manageiq-ui-classic/pull/9598 removed this function and others, please rebase / replace with the new function., sorry 😢